app/views/tmdb/search.html.erb

```diff
@@ -1,29 +1,37 @@
 <% content_for(:title, "Search by Movie Title") %>
 
-<% if !@movie_title.present? %>
+<% if @movie_title.blank? %>
```

Comment:
I prefer to use a positive expression instead of a double negative when possible.

Reply:
totes!